### ISSUE TITLE
Make `plot_learning_curve` graphs for any metric in History object

### ISSUE THREAD

**Issue by aiqc** [aiqc/AIQC#66]
> Difficulty: ★★☆☆☆

## Background
Training metrics go into `Result.history`. And `Result.plot_learning_curve()` plots these metrics.

## Problem
Right now it only plots for loss and, if `analyis_type=='classification'`, accuracy metrics. Regression analyses do not have an accuracy.

## Solution
Check what metrics are in the history object and make a graph for each metric. In Keras, the `History` callback and the `model.compile(metrics = ['accuracy'])`. In PyTorch, the history object is user-defined.

**Comment by aiqc** [comment]
solved
